pkg/cpi/mockingUtils.go

```diff
@@ -41,3 +56,27 @@ func GetCPIFunctionMockResponse(functionName, testType string) (*http.Response,
 		return &res, errors.New("Service not Found")
 	}
 }
+
+//GetEmptyHTTPResponseBody -Empty http respose body
+func GetEmptyHTTPResponseBody() (*http.Response, error) {
+	res := http.Response{
+		StatusCode: 202,
+		Body:       ioutil.NopCloser(bytes.NewReader([]byte(``))),
+	}
+	return &res, nil
+}
+
+//GetNegativeCaseHTTPResponseBody -Negative case http respose body
+func GetNegativeCaseHTTPResponseBody() (*http.Response, error) {
```

Comment:
This function is not needed by the changes of this PR.

Reply:
removed